xml/networking-dcn_core.xml

```diff
@@ -9,19 +9,18 @@
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="DCN_core">
  <title>Bringing up &productname; with the DCN Core Driver</title>
  <para>
-  Bringing up a &kw-hos; cloud with the DCN core plugin is identical to
-  <xref linkend="DCN_ml2"/>, with the exception of the following required
+  Bringing up a &kw-hos; cloud with the DCN core plugin requires the following
```

Comment:
This makes no sense anymore without the original section being present. I guess you might need a technical expert to merge those two sections correctly though.